catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/source/WfsFilterDelegateTest.java

```diff
@@ -0,0 +1,1665 @@
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXmlLiteral, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToDate()
+      throws JAXBException, SAXException, IOException {
+
+    whenPropertiesBooleanType();
+    WfsFilterDelegate delegate = createDelegate();
+
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, date);
+    assertXMLEqual(propertyGreaterThanOrEqualToXmlDate, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToInt()
+      throws JAXBException, SAXException, IOException {
+    whenPropertiesIntegerType();
+    WfsFilterDelegate delegate = createDelegate();
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, 1);
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXml, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToShort()
+      throws JAXBException, SAXException, IOException {
+    whenPropertiesIntegerType();
+    WfsFilterDelegate delegate = createDelegate();
+    short literal = 1;
+
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, literal);
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXml, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToLong()
+      throws JAXBException, SAXException, IOException {
+    whenPropertiesLongType();
+    WfsFilterDelegate delegate = createDelegate();
+    long literal = 1L;
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, literal);
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXml, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToFloat()
+      throws JAXBException, SAXException, IOException {
+    whenPropertiesLongType();
+    WfsFilterDelegate delegate = createDelegate();
+    float literal = 1.0F;
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, literal);
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXmlDecimal, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanOrEqualToDouble()
+      throws JAXBException, SAXException, IOException {
+    whenPropertiesLongType();
+    WfsFilterDelegate delegate = createDelegate();
+    double literal = 1.0;
+    FilterType filter = delegate.propertyIsGreaterThanOrEqualTo(MOCK_PROPERTY, literal);
+
+    assertXMLEqual(propertyGreaterThanOrEqualToXmlDecimal, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsLessThanString() throws JAXBException, SAXException, IOException {
+    whenPropertiesBooleanType();
+    WfsFilterDelegate delegate = createDelegate();
+
+    FilterType filter = delegate.propertyIsLessThan(MOCK_PROPERTY, LITERAL);
+
+    assertXMLEqual(propertyLessThanXmlLiteral, marshal(filter));
+  }
+
+  @Test
+  public void testPropertyIsLessThanDate() throws JAXBException, SAXException, IOException {
+    whenPropertiesBooleanType();
+    WfsFilterDelegate delegate = createDelegate();
```

Comment:
✏️  as a nit: we still could extract some of this creation out into a separate method